src/main/java/com/direwolf20/buildinggadgets/client/gui/ModeRadialMenu.java

```diff
@@ -16,6 +16,8 @@
 import com.direwolf20.buildinggadgets.common.network.PacketToggleConnectedArea;
 import com.direwolf20.buildinggadgets.common.network.PacketToggleFuzzy;
 import com.direwolf20.buildinggadgets.common.network.PacketToggleMode;
+import com.direwolf20.buildinggadgets.common.tools.BuildingModes;
```

Comment:
the tools package isn't the right place for these ( sorry )

Reply:
Do you have any suggestions of where to put these? What does `tools` imply here, does it mean gadget's logic or all external stuff or something else?

Reply:
You propably should create a new package...
common.building maybe? (I don't really have any good suggestions either)